doc/contributing.md

```diff
@@ -479,6 +482,74 @@ make check   # Runs black, blackdoc, docformatter, flake8 and isort (in check mo
 make lint    # Runs pylint, which is a bit slower
 ```
 
+### Wrapping a new GMT module
+
+Wrapping a new GMT module in PyGMT is usually a big task, which will progress
+quicker and smoother if done in **small, maneable chunks**. This section
+gives an overview of the specific tasks involved in wrapping a new module.
+
+1. Create a ['Wrapper for `<module-name>`' feature request issue](#request-wrapper-for-gmt-module). 
+2. Open a ['Wrap `<module-name>`' initial feature implementation PR](#initial-feature-implementation).
+3. Open an ['Add missing aliases to `<module-name>`' documentation PR](#add-missing-aliases).
+4. Open a 'support additional functionality in module' PR (optional).
+5. Add ['Gallery example for module' documentation PR](#contributing-gallery-plots).
+6. Add ['Tutorial for module' documentation PR](#contributing-tutorials) (optional).
+
+These steps will be tracked in the 'Wrapper for `<module-name>`' issue and the
+['wrapping GMT modules'](https://github.com/GenericMappingTools/pygmt/projects/9)
+project board. The pull requests can be split between multiple contributors and
```

Comment:
I think we should either track the module updates in a project board (my preference) or an issue, but not both.

Reply:
Do you think we should abandon the 'Wrapper for `<module-name>`' issues entirely or keep the issues/issue-template and just move the checklist to the project board? I think the main benefit of the issues is that from my understanding any GitHub user can open/comment on an issue whereas writing on the project board requires special permissions.

Reply:
I think the project board works better for tracking the progress of wrapping a module, and that having a separate issue for wrapping the function is a bit redundant. I see what you mean about the benefits of issues being open to everyone, but my thought is that wrapping modules is a pretty routine process and doesn't need to first be raised as an issue (unlike a bug or a feature request). We can add some/all of the GMT modules to the project board to track their process. Users who don't have write permission are still free to open up an issue requesting a module/feature (or a pull request! :crossed_fingers:) as a way to let us know that a certain feature would be useful to them.

Reply:
> the benefits of issues being open to everyone

IMHO, issues are also more visible than project boards, so that we may have more potential contributors helping wrap new modules and add more aliases.